segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/ExtendedS3IntegrationTest.java

```diff
@@ -39,7 +41,7 @@
 public class ExtendedS3IntegrationTest extends StreamSegmentStoreTestBase {
     //region Test Configuration and Setup
 
-    private static final int BOOKIE_COUNT = 3;
+    private static final int BOOKIE_COUNT = 1;
```

Comment:
Not 100% related, but I was in the file and had to debug it a few times and realized that it was 3 instead of 1. For unit tests and in-process integration tests, there's no need in having more than one bookie - it's just wasting resources and making the test slower.

Reply:
reverted